docs/newbs_flashing.md

```diff
@@ -72,6 +78,163 @@ Click the `Flash` button in QMK Toolbox. You will see output similar to the foll
 *** Clueboard - Clueboard 66% HotSwap connected -- 0xC1ED:0x2390
 ```
 
+## Flash your Keyboard from the Command Line
+
+First thing you'll need to know is which bootloader that your keyboard uses.  There are four main bootloaders that are used, usally. Pro-Micro and clones use CATERINA, and Teensy's use Halfkay, OLKB boards use QMK-DFU, and other atmege32u4 chips use DFU. 
```

Comment:
usally -> usually

Reply:
Curse my spellchecker.

Fixed. And a couple more not already spotted